hack/lint-gofmt.sh

```
@@ -6,7 +6,7 @@ WMCO_ROOT=$(dirname "${BASH_SOURCE}")/..
 
 GO_VERSION=($(go version))
 
-if [[ -z $(echo "${GO_VERSION[2]}" | grep -E 'go1.25') ]]; then
+if [[ -z $(echo "${GO_VERSION[2]}" | grep -E 'go1.26') ]]; then
```

Shellcheck (0.11.0)

[style] Use ! grep -q instead of comparing output with [ -z .. ].

(SC2143)